src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue30052.cs

```diff
@@ -0,0 +1,20 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue30052 : _IssuesUITest
+{
+	public Issue30052(TestDevice testDevice) : base(testDevice) { }
+
+	public override string Issue => "Right-To-Left (RTL) alignment is not applied to Editor placeholder";
+
+	[Test]
+	[Category(UITestCategories.Editor)]
+	public void RTLShouldBeAppliedToPlaceholers()
+	{
+		App.WaitForElement("Editor");
+		VerifyScreenshot();
```

Comment:
Pending snapshots already available on the latest build.
![image](https://github.com/user-attachments/assets/77e8a434-6bcf-43d1-94ad-50f63efbb3f8)
Could you commit the images?